resources/assets/enderio/config/SAGMillRecipes_Core.xml

```diff
@@ -686,22 +722,21 @@ addOreDictionaryTooltips=true
   </recipeGroup>
 
 
-  <recipeGroup name="Applied Energistics">
-
+  <recipeGroup name="Applied Energistics 2">
 
     <recipe name="ChargedCertusQuartzOre" energyCost="2400">
       <input>
-        <itemStack  modID="appliedenergistics2" itemName="tile.OreQuartzCharged"  />
+        <itemStack modID="appliedenergistics2" itemName="tile.OreQuartzCharged" />
       </input>
       <output>
-        <itemStack modID="appliedenergistics2" itemName="item.ItemMultiMaterial" itemMeta="1"/>
+        <itemStack modID="appliedenergistics2" itemName="item.ItemMultiMaterial" itemMeta="1" />
```

Comment:
Still seems low imo. Maybe do some testing with the ore and mining, see what a good average value is, and remember that the SAG Mill typically gives a *bonus* over normal processing.